src/stores/businessBootstrap.ts

```diff
@@ -126,6 +210,29 @@ export const useBcrosBusinessBootstrap = defineStore('bcros/businessBootstrap',
     trackUiLoadingStop('boostrapBusinessLoading')
   }
 
+  const loadPendingFiling = () => {
```

Comment:
does this have a call to backend ? if so it could be good to have async and await